package.json

```diff
@@ -174,6 +176,7 @@
     "sass-true": "6.0.1",
     "sinon": "12.0.1",
     "snyk": "1.1291.0",
+    "style-dictionary": "3.9.2",
```

Comment:
Released last month! That's great. I can look into migrating to 4.x

Reply:
@mahoneycm just a quick look might be needed to understand LOE. The _actual_ migration might be out-of-scope.

Reply:
Looks like it's going to be a moderate LOE. I toyed with the initial migration for a few minutes and ran into ESM import issues. It could be a candidate for next steps as we define the timeline for tokens.

Reply:
Looks like style-dictionary `4.x` requires ESM format and we likely won't be able to upgrade until USWDS supports ESM.